contact.html

```diff
@@ -1,11 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="index.css" /> 
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
-</head>
+   </head>
 <body>
+    <div id="bar">
     
+       <a href="index.html" target="_blank">Home</a>
+    
+    </div>
+    <h2>Contact Me</h2>
+    <fieldset>
+        <legend>Register</legend>
+        <form align = "center">
+            <label>Name</label>
+            <input type="text">
+            <br><br>
+            <label>Email</label>
+            <input type="text">
+            <br><br>
+            <label>Phone</label>
+            <input type="tel">
+            <br><br>
+            <textarea rows="4" cols="50" name="comment" form="usrform">
+                Enter message here...</textarea><br><br>
+            <input type="submit">
+        </form>
+    </fieldset>
+    <!--<div id="bgimg" align="center">
+        <br>
+        <img id="img" src="/image/anu.jpeg">
+        <h2 style="text-align:center;line-height: 5px;">Anuradha Sathiyachalam</h2>
+        <h3 style="text-align:center;line-height: 5px;">Dallas, TX</h3>
+        <h3 style="text-align:center;line-height: 5px;">Email: anoosathyee@gmail.com</h3>
+        <h3 style="text-align:center;line-height: 5px;">Phone: +1(972)510-8560</h3>
+
+         <a href="https://www.linkedin.com/in/anuradha-sathiyachalam-9aa49265/" target="_blank">LinkedIn</a>&nbsp;&nbsp;|&nbsp;&nbsp;
+         <a href="https://github.com/AnooSathyee" target="_blank">GitHub</a>
+    </div>
```

Comment:
Avoid leaving code in your file which is commented out.  Someone attempting to work with your code in the future may inadvertently uncomment those lines, introducing issues you intended to avoid.